gmailservice/gmailservice_test.go

```diff
@@ -217,10 +225,10 @@ func TestDownloadFullMessages(t *testing.T) {
               successes: 0,
             },
             finalAdvance: results{
-              failures:  9,
-              successes: 0,
+              failures:  6,
```

Comment:
This test was to test what happens when the API never returns success. Now it seems to be testing the same as the test case above it?

Reply:
ah, ok.  I didn't really understand what was intended.  Fixed!